styles/attention.css

```diff
@@ -0,0 +1,24 @@
+.attention-container {
+    width: 100%;
+    background-color: #f77462;
+    box-sizing: border-box;
+    padding: 55px 0 51px 0;
+}
+
+.attention__title, .attention__prh {
+    color: white;
```

Comment:
You need to be consistent in colors. Advised schemes of declaring colors are: hex notation (6 digits, all lower-case); rgba, hsla. Avoid using shorthands like `white` or `cyan`, using rgb and hsl without transparency parameter (in this case it's better to opt for hex `#ffffff` notation).